README.md

```diff
@@ -24,6 +24,49 @@ Include it in your Cargo.toml:
 inotify = "*"
 ```
 
+And here's a little example:
+```Rust
+#![feature(path)]
+
+extern crate inotify;
+
+use inotify::INotify;
+use inotify::ffi::*;
+use std::path::Path;
+
+fn main() {
+    let mut ino = INotify::init().unwrap();
+
+    ino.add_watch(Path::new("/home"), IN_MODIFY | IN_CREATE | IN_DELETE).unwrap();
+    loop {
+        let events = ino.wait_for_events().unwrap();
+
+        for event in events.iter() {
+            if event.is_create() > 0 {
```

Comment:
Now it's wrong :)
`is_create` (and the other `Event` functions) return bool, so the `> 0` won't compile.